test/LeftNav_test.jsx

```diff
@@ -0,0 +1,197 @@
+import assert from "assert";
+import React from "react";
+import sinon from "sinon";
+import {shallow} from "enzyme";
+import {LeftNav} from "../src";
+
+describe("LeftNav", function LeftNavTest() {
```

Comment:
Apparently if you want to use `this` to share context between `before` and `it` blocks, you have to have a named function somewhere in your test suite hierarchy. Otherwise you get an error from mocha that `this` is undefined.